src/OpenTelemetry/Trace/TracerProviderExtensions.cs

```diff
@@ -43,6 +43,34 @@ public static TracerProvider AddProcessor(this TracerProvider provider, BaseProc
             return provider;
         }
 
+        public static bool ForceFlush(this TracerProvider provider, int timeoutMilliseconds = Timeout.Infinite)
```

Comment:
Please add inline docs here since this is public API (follow what we have at line 74).